templates/zerver/help/saml-authentication.md

```diff
@@ -1,166 +1,196 @@
 # SAML authentication
 
-Zulip supports using SAML authentication for single sign-on, both when
-self-hosting or on the Zulip Cloud Plus plan.
+{!admin-only.md!}
 
-This page documents details on how to set up SAML authentication with
-Zulip with various common SAML identity providers. Once you have set up
-SAML, you may also want to consider configuring [SCIM](/help/scim).
+Zulip supports using SAML authentication for single sign-on, both for Zulip
+Cloud and self-hosted Zulip servers.
```

Comment:
Here was say that SAML is supported on Zulip Cloud - which is true but with the Plus plan caveat, that we have a warning for below? Should the warning be perhaps right above or right below this sentence, to make sure there is no confusion?

Reply:
I dunno, I didn't love the visual effect of having two warning banners in a row, and there is a secondary mention of the Plus plan in the first point of the setup instructions, so I think it would be hard to miss?

@timabbott what do you think?

Reply:
I think we could just say, for example, "both for Zulip Cloud on the Plus plan and self-hosted Zulip servers".

Reply:
After some reflection, I think it's reasonable to not complicate this sentence; the banner is on screen when you're reading this.